.github/workflows/ci.yaml

```
@@ -105,14 +109,11 @@ jobs:
       - name: Checkout sources
         uses: actions/checkout@v3
 
-      - name: Install stable toolchain
-        uses: actions-rs/toolchain@v1
+      - name: Install Rust ${{ env.rust_clippy }}
+        uses: dtolnay/rust-toolchain@master
         with:
-          profile: minimal
-          toolchain: stable
-          override: true
-          components: rustfmt, clippy
-      - uses: Swatinem/rust-cache@v1
+          toolchain: ${{ env.rust_fmt_clippy }}
```

similarly, this also references an env var that doesn't exist (and the build is failing due to it): https://github.com/tokio-rs/console/actions/runs/4766006287/jobs/8472458613?pr=413
```suggestion
          toolchain: ${{ env.RUST_FMT_CLIPPY }}
```